skyportal/tests/frontend/test_favorites_page.py

```diff
@@ -26,8 +26,16 @@ def test_add_remove_favorites(driver, user, public_source):
     # little triangle you push to expand the table
     driver.click_xpath("//*[@id='expandable-button']")
 
+    driver.wait_for_xpath(
+        f"//*[contains(@data-testid, 'favorites-text-include_{public_source.id}')]",
+        timeout=10,
+    )
+
     # click to un-save the source as favorite
-    driver.click_xpath(f'//*[@data-testid="favorites-text-include_{public_source.id}"]')
+    driver.click_xpath(
+        f'//*[@data-testid="favorites-text-include_{public_source.id}"]',
+        scroll_parent=True,
```

Comment:
I've had to remove a few `scroll_parent=True` statements from various test (I can't remember which ones) but it may be a difference in browser from the GA. Are you sure this actually helps the test pass?

Reply:
Good point, removing for now.